src/libraries/System.Private.CoreLib/src/System/DefaultBinder.cs

```diff
@@ -395,7 +397,7 @@ internal partial class DefaultBinder : Binder
 
             // If the parameters and the args are not the same length or there is a paramArray
             //  then we need to create a argument array.
-            ParameterInfo[] parameters = candidates[currentMin]!.GetParametersNoCopy();
+            ParameterInfo[] parameters = bestMatch.GetParametersNoCopy();
```

Comment:
No reason to keep derefing the array, we know what it is above (miniscule performance increase _possible_).